src/LfMergeBridge/LanguageForgeMakeCloneActionHandler.cs

```diff
@@ -178,7 +180,8 @@ void IBridgeActionTypeHandler.StartWorking(IProgress progress, Dictionary<string
 
 			var user = options.ContainsKey(LfMergeBridgeUtilities.user) ? options[LfMergeBridgeUtilities.user] : null;
 
-			FinishClone(progress, ref somethingForClient, cloneBase, actualClonePath, options[LfMergeBridgeUtilities.fdoDataModelVersion], user, DeleteRepoIfNoSuchBranch(options));
+			var desiredBranchName = FlexBridgeConstants.FlexBridgeDataVersion + "." + options[LfMergeBridgeUtilities.fdoDataModelVersion];
```

Comment:
It seems to me this (and similar places where we calculate the branch name) should be refactored so that there is a method that takes a model version and returns the branch name, kind of the opposite of  `GetModelVersionFromBranchName`. This could maybe be added to `IUpdateBranchHelperStrategy`.